src/site/stages/build/drupal/graphql/navigation-fragments/facilitySidebar.nav.graphql.js

```diff
@@ -9,6 +9,14 @@ const { camelize } = require('./../../../../../utilities/stringHelpers');
 const FACILITY_MENU_NAMES = [
   'pittsburgh-health-care',
   'va-altoona-health-care',
+  'va-butler-health-care',
+  'va-coatesville-health-care',
+  'va-altoona-health-care',
```

Comment:
'va-altoona-health-care' is in there twice.

Reply:
it's alpha sorted now that the dupe is removed.